$OpenBSD: patch-src_OScopeCtrl_cpp,v 1.3 2019/01/20 22:02:02 juanfra Exp $

Fixes this crash:
"Assertion failed: ./src/gtk/dc.cpp:wxPaintDCImpl:276: Assertion 'cr' failed.
using wxPaintDC without being in a native paint event"

http://forum.amule.org/index.php?topic=21405.0
http://forum.amule.org/index.php?topic=21383.15

Index: src/OScopeCtrl.cpp
--- src/OScopeCtrl.cpp.orig
+++ src/OScopeCtrl.cpp
@@ -522,7 +522,7 @@ void COScopeCtrl::InvalidateCtrl(bool bInvalidateGraph
 	bRecreateGraph |= bInvalidateGraph;
 	bRecreateGrid |= bInvalidateGrid;
 	// It appears the timerRedraw logic screws up Mac, disable it there
-#ifndef __WXMAC__
+#if 0
 	// To prevent startup problems don't start timer logic until
 	// a native OnPaint event has been generated.
 	if (m_onPaint) {
